src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Areas/Identity/Pages/Account/ExternalLoginTests.cs

```diff
@@ -0,0 +1,576 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Castle.Core.Internal;
```

Comment:
This import looks a bit suspect. I think it's coming from using `IsNullOrEmpty` rather than our own.

Reply:
Good spot!  Changed to use ours